types/kaluma/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "private": true,
+    "name": "@types/kaluma",
+    "version": "1.1.9999",
+    "nonNpm": true,
+    "nonNpmDescription": "Kaluma - A tiny JavaScript runtime for RP2040 (Raspberry Pi Pico)",
+    "projects": [
+        "https://github.com/kaluma-project/kaluma"
+    ],
+    "devDependencies": {
+        "@types/kaluma": "workspace:."
+    },
+    "owners": [
+        {
+            "name": "Jason Tremper",
+            "githubUsername": "jt000"
+        },
+        {
+            "name": "Minkyu Lee",
+            "githubUsername": "niklauslee"
```

Comment:
would be nice to get this user ACK to be included as the owner (I understand this is a lead/one of contributors to a project)

Reply:
Thanks! I'll take them off for now & re-add after I get confirmation